common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -345,29 +345,20 @@ def answer_problem(self, correct):
             self.problem_page.click_choice("choice_1")
 
     @attr('shard_7')
-    def test_can_show_hide_answer(self):
+    def test_can_show_answer(self):
         """
-        Scenario: Verifies that show/hide answer button is working as expected.
+        Scenario: Verifies that show answer button is working as expected.
```

Comment:
It is odd that some problem tests are in this file/class, and others are in test_lms_courseware.py. "Show Answer" is being tested in both places. We need to pick a single location for these tests and remove the duplication.